### PR TITLE
Translate severity levels and module names in result page

### LINKED ISSUE #86: Translation of categories, type of errors...
Categories like NAMESERVER, ADDRESS or levels of message WARNING, ERROR are not translated... Is it something that we should consider ???

### PR DESCRIPTION
**Pull request by hannaeko**
## Purpose

Translate severity levels and module names

## Context

Fixes #86

## Changes

* Translate severity levels
* Translate modules names

## How to test this PR

* Load a test result in a language that is not English and have the translation done
* Check that the module names are translated
* Check that the tooltip of the severity levels in module and test case heading icon are translated
* Check that the severity levels in test case messages are translated

### CODE REVIEW

**Review by marc-vanderwal — DISMISSED** [review]

File: src/app/components/result/result.component.ts [hunk 1]
```diff
@@ -32,7 +32,26 @@ export class ResultComponent implements OnInit, OnDestroy {
...
+    'system': $localize `System`,
+    'basic': $localize `Basic`,
+    'address': $localize `Address`,
+    'connectivity': $localize `Connectivity`,
+    'consistency': $localize `Consistency`,
+    'delegation': $localize `Delegation`,
+    'dnssec': $localize `DNSSEC`,
+    'nameserver': $localize `Nameserver`,
+    'syntax': $localize `Syntax`,
+    'zone': $localize `Zone`,
```
Comment (matsduf) [inline src/app/components/result/result.component.ts:53]: This is a list of the current modules. What is a new module name comes, e.g. the name of a custom module?

Comment (marc-vanderwal) [reply src/app/components/result/result.component.ts:53]: That’s a limitation of the current design. Shouldn’t these module names be moved over to Zonemaster::Engine, eventually?

Comment (matsduf) [reply src/app/components/result/result.component.ts:53]: If custom module names just show up untranslated, it is fine. If they disappear, it is not fine

I think we should consider moving it to Engine later but it is fine to have it in GUI for now.

**Review by matsduf — DISMISSED** [review]
My approval is based on the assumption that names of custom modules are just untranslated, and not creating any other issue

**Comment by hannaeko** [comment]
> My approval is based on the assumption that names of custom modules are just untranslated, and not creating any other issue

This was not the case and should now be fixed.

> That’s a limitation of the current design. Shouldn’t these module names be moved over to Zonemaster::Engine, eventually?

I implemented this based on the decision taken in #86 to only translate in Gui but I guess this issue is a bit now and things have changed since ^^

**Comment by matsduf** [comment]
> I implemented this based on the decision taken in #86 to only translate in Gui but I guess this issue is a bit old now and things have changed since ^^

I think that is fine for now. We can take the next step later.

**Review by matsduf — APPROVED** [review]

**Comment by matsduf** [comment]
@tgreenx and @marc-vanderwal, please re-review.

**Review by marc-vanderwal — APPROVED** [review]

**Pull request merged**